vite.config.ts

```diff
@@ -36,5 +37,20 @@ export default defineConfig({
     coverage: {
       reporter: ['text', 'lcov', 'html']
     }
+  },
+  preview: {
+    port: 8099,
+    proxy: {
+      '/api': {
+        target: 'http://localhost:9091/',
+        changeOrigin: true,
+        secure: false,
+        xfwd: true
+      }
+    },
+    headers: {
+      'Content-Security-Policy':
```

Comment:
This looks like duplication as we already have a similar configuration in the NGINX config. Is it really necessary to do that when vite is used?

Reply:
That's for local development. NGINX is used for production. I added this to test locally if the code works with the CSP via 'npm run preview'